Jumpscale/clients/blockchain/tfchain/TFChainClientFactory.py

```diff
@@ -23,6 +24,11 @@ def _init(self):
         self._explorer_client = TFChainExplorerClient()
         self._types_factory = TFChainTypesFactory()
         self._error_types = ErrorTypes()
+        self._time = TFChainTime
+
+    @property
+    def time(self):
```

Comment:
Should this be in the root of the factory, or in the TFChainCapacity class ?

Reply:
I'd rather have it here like jumpscale_lib and not have it tied to initializing a wallet instance which the capacity class needs.

Reply:
ok